### PR TITLE
api: Http only cookie setting

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: ci/tools.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ import (
 	_ "github.com/cratonica/2goarray"
 	_ "github.com/envoyproxy/protoc-gen-validate"
 	_ "github.com/golang/mock/mockgen"
+	_ "github.com/onsi/ginkgo/ginkgo"
```
Comment (nfuden) [inline ci/tools.go:24]: This seems unrelated. Why do we now need to import ginkgo for side effects?

Comment (yuval-k) [reply ci/tools.go:24]: good question; make codegen didn't work for me without it. it could be because i'm using go 1.17

**Review by nfuden — APPROVED** [review]
